mega-linter-runner/lib/options.js

```diff
@@ -122,7 +122,7 @@ module.exports = optionator({
       description: "Upgrade local repository MegaLinter configuration",
     },
     {
-      option: "containername",
+      option: "container-name",
```

Comment:
Adding `alias: "containername"` will maintain backward compatibility:
```suggestion
      option: "container-name",
      alias: "containername",
```

Reply:
@nvuillam, I think this can now be merged in v6 thanks to @dmitri-trofimov's excellent suggestion.